ci/pipeline.yaml

```diff
@@ -136,6 +136,11 @@ resources:
       tag: latest
       username: iamapikey
       password: ((concourse-icr-containers-public.password))
+  - name: preflight
+    type: github-release
```

Comment:
Don't we need to define the type in the `resource_types` section?

Reply:
No. It works. I've tested it.